packages/next/src/lib/verify-typescript-setup.ts

```diff
@@ -53,6 +54,7 @@ export async function verifyTypeScriptSetup({
   disableStaticImages: boolean
   hasAppDir: boolean
   hasPagesDir: boolean
+  nextConfig?: { experimental?: { typedRoutes?: boolean } }
```

Comment:
Can we still use `NextCompleteConfig` or at least the `NextConfig` type if it can't be complete?